src/analysis/modals/CloudEnvironmentModal.ts

```diff
@@ -463,13 +465,17 @@ export const CloudEnvironmentModal = ({
             href:
               app &&
               (cloudProvider === cloudProviderTypes.AZURE
-                ? app?.proxyUrls['cbas-ui']
+                ? isFeaturePreviewEnabled(WORKFLOWS_TAB_AZURE_FEATURE_ID)
+                  ? Nav.getLink('workspace-workflows-app', { namespace, name })
```

Comment:
Hmm... I strongly dislike nested ternary expressions. Can we use a `Utils.cond` for this logic to make it clearer which result is from which conditional?